webpack/assets/javascripts/react_app/components/ReportTemplate/Inputs/ReportAutoComplete.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FieldLevelHelp } from 'patternfly-react';
+
+import Form from '../../common/forms/CommonForm';
+import AutoComplete from '../../AutoComplete';
+
+const ReportAutoComplete = ({
+  label,
+  resourceType,
+  id,
+  initialQuery,
+  initialError,
+  info,
+  isRequired,
+  url,
+}) => (
+  <Form
+    label={label}
+    error={initialError}
+    required={isRequired}
+    tooltipHelp={info && <FieldLevelHelp content={<div>{info}</div>} />}
+  >
+    <AutoComplete
+      id={`report-input-${id}`}
```

Comment:
everywhere where you use report-input, it should rather be template-input, not that these inputs are generic for all template kinds

Reply:
makes sense